### PR TITLE
Make link to SDK reference more prominent

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

✅ content/spin/v2/language-support-overview.md

- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-12-15 at 09 05 21

- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

**Comment by tpmccallum** [comment]
These links to the SDKs answer a lot of questions (how features are implemented in each specific language) i.e. resolve questions about syntax case `max-tokens`, max_tokens`, `maxTokens`, `MaxTokens`. 
Repeating this information for the sake of surfacing the information would create redundancy. So instead this will make this information a bit more prominent and accessible.

**Comment by itowlson** [comment]
What is the little image @tpmccallum?  It's not showing in the PR review tab, and I can't quite make it out on the screenshot.

**Comment by tpmccallum** [comment]
It is a document symbol, just to make it stand out a bit.
Screenshot 2023-12-15 at 09 14 55

File: content/spin/v2/language-support-overview.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Rust"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html)
+**[📄 Visit the Rust Spin SDK reference documentation](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html) to see specific modules, functions, variables and syntax relating to the following Rust features.**
```
Comment (dicej) [inline content/spin/v2/language-support-overview.md:16]:
```suggestion
**[📄 Visit the Rust Spin SDK reference documentation](https://docs.rs/spin-sdk/latest/spin_sdk/) to see specific modules, functions, variables and syntax relating to the following Rust features.**
```
This is probably the more "official" place to see the Rust docs now that we're publishing to crates.io,

**Comment by tpmccallum** [comment]
Happy to loose it, or perhaps try using a callout box instead.

**Review by dicej — APPROVED** [review]

File: content/spin/v2/language-support-overview.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Python"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/spin-python-sdk/)
+**[📄 Visit the Python Spin SDK reference documentation](https://fermyon.github.io/spin-python-sdk/) to see specific modules, functions, variables and syntax relating to the following Python SDK.**
```
Comment (itowlson) [inline content/spin/v2/language-support-overview.md:62]:
```suggestion
**[📄 Visit the Python Spin SDK reference documentation](https://fermyon.github.io/spin-python-sdk/) to see the modules, types, functions, and variables in the Python SDK.**
```

(Although personally I find that extra clause a bit redundant.  It's kind of "visit the Python SDK docs to see what's in the Python SDK."  And that enumerating the various kinds of things detracts from the message rather than adding to it.  But YMMV!)

File: content/spin/v2/language-support-overview.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "TinyGo"}}
 
-[Visit the Spin SDK reference documentation.](https://pkg.go.dev/github.com/fermyon/spin/sdk/go/v2)
+**[📄 Visit the TinyGo Spin SDK reference documentation](https://pkg.go.dev/github.com/fermyon/spin/sdk/go/v2) to see specific modules, functions, variables and syntax relating to the following TinyGo SDK.**
```
Comment (itowlson) [inline content/spin/v2/language-support-overview.md:85]:
```suggestion
**[📄 Visit the TinyGo Spin SDK reference documentation](https://pkg.go.dev/github.com/fermyon/spin/sdk/go/v2) to see the modules, types, functions, and variables in the TinyGo SDK.**
```

File: content/spin/v2/language-support-overview.md (hunk 1 above)
Comment (itowlson) [inline content/spin/v2/language-support-overview.md]: For me the document symbol seemed a bit blurry and didn't draw the eye.  Might this look better with something bolder like a BIG HONKIN' ARROW or something like that?

Comment (itowlson) [inline content/spin/v2/language-support-overview.md]:
```suggestion
**[📄 Visit the Rust Spin SDK reference documentation](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html) to the modules, types, functions, and macros relating to the following Rust features.**
```

File: content/spin/v2/language-support-overview.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "TypeScript"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/spin-js-sdk/)
+**[📄 Visit the JS/TS Spin SDK reference documentation](https://fermyon.github.io/spin-js-sdk/) to see specific modules, functions, variables and syntax relating to the following TS/JS features.**
```
Comment (itowlson) [inline content/spin/v2/language-support-overview.md:39]:
```suggestion
**[📄 Visit the JS/TS Spin SDK reference documentation](https://fermyon.github.io/spin-js-sdk/) to see the modules, functions, types, and variables relating to the following TS/JS features.**
```

File: content/spin/v2/language-support-overview.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Rust"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html)
+**[📄 Visit the Rust Spin SDK reference documentation](https://docs.rs/spin-sdk/latest/spin_sdk/) to see specific modules, functions, variables and syntax relating to the following Rust features.**
```
Comment (itowlson) [inline content/spin/v2/language-support-overview.md:16]:
```suggestion
**[📄 Visit the Rust Spin SDK reference documentation](https://docs.rs/spin-sdk/latest/spin_sdk/) to see the modules, types, functions, and macros relating to the following Rust features.**
```

Comment (itowlson) [inline content/spin/v2/language-support-overview.md:16]: For me the document symbol is blurry and hard to parse - it doesn't really draw my attention.  How might it look with something more dramatic like a BIG HONKIN' ARROW or something like that?

File: content/spin/v2/language-support-overview.md (hunk 2 above)
Comment (itowlson) [inline content/spin/v2/language-support-overview.md:62]: There's a style shift where some sections relate to "the following <language> features" and some to "the <language> SDK" - is that intentional?

**Review by itowlson — COMMENTED** [review]
I am not sure quite what happened but a couple of comments disappeared when the document updated.  The count seems to imply they still exist but I can't see them!  So if there are duplicated loose comments then I apologise.

I am all in favour of this but prefer (a) punchier phrasing, no need to list all the various syntactic elements of each SDK and (b) could we try with a simpler icon and see if that works please?  Thanks
